easybuild/easyconfigs/a/Armadillo/Armadillo-11.4.3-foss-2022b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'Armadillo'
+version = '11.4.3'
+
+homepage = 'http://arma.sourceforge.net/'
```

Comment:
I don't see a reason to not use `https` here?

Reply:
fixed with [1242e25](https://github.com/easybuilders/easybuild-easyconfigs/pull/17190/commits/1242e25ff7f0d415a9b0aa39542f7f59c8b0566d)